src/HotChocolate/Data/src/Data/Filters/Types/IdOperationFilterInputType.cs

```diff
@@ -0,0 +1,75 @@
+using HotChocolate.Configuration;
+using HotChocolate.Internal;
+using HotChocolate.Types;
+using HotChocolate.Types.Descriptors.Definitions;
+
+namespace HotChocolate.Data.Filters
+{
+    public class IdOperationFilterInputType<TEntityType, TIdType>
```

Comment:
To initialise the ID serialiser I needed the entity type and the type of the id.
If you have a better idea let me know.
We could push it down with a type interceptor, but in the end, the user has to define the ID operation types anyway by hand, because we do not yet have a way to infere the ID property from the dotnet type.